### ISSUE TITLE
Handles of custom gizmos rendered as only a single pixel

### ISSUE THREAD

**Issue by blackears** [godotengine/godot#75680]
### Godot version

v4.0.stable.official [92bee43ad]

### System information

Windows 10, Compatability

### Issue description

I stated exploring using gizmos for a plugin I'm working on.  The handles that are rendered are so tiny, they look like they're only a single pixel big.  I double checked with Godot 3.5.2 and it's equivalent of the plugin renders handles at a visible size.  I tried using a custom texture too, but that id not help.  The code I'm using here is pretty close to what appears on the Godot website as a tutorial  Maybe I missed something, but I think this is a problem with the engine.

### Steps to reproduce

Write a simple plugin using the code described on this page:
https://docs.godotengine.org/en/stable/tutorials/plugins/editor/3d_gizmos.html

The handles will be drawn so tiny you won't be able to see them. (You can still click and drag on them though).

### Minimal reproduction project

[simpleGizmoTest.zip](https://github.com/godotengine/godot/files/11156862/simpleGizmoTest.zip)

**Comment by Zireael07** [comment]
IIRC Compatibility renderer (GLES 3) has no concept of line width, leading to this issue.

**Comment by Calinou** [comment]
- Duplicate of https://github.com/godotengine/godot/issues/74005. This is fixed by https://github.com/godotengine/godot/pull/73966, which will be in 4.0.2.

**Comment by blackears** [comment]
I'm still getting the issue in 4.0.2.

It might be related to my graphics card.  I can't run Vulkan and am relying on whatever Godot is using to run on older hardware.  My graphics card is a FirePro™ W2100.



![image](https://user-images.githubusercontent.com/6724608/230093775-6c95ea61-fc93-4b1e-8c1a-794b908eec8a.png)

**Comment by Calinou** [comment]
AMD OpenGL on Windows has a tendency not to support line width and point size, so I doubt this can be fixed unless we make gizmos use billboarded quads.

**Comment by blackears** [comment]
Wouldn't that be an option?  At least for compatibility with older systems?

**Comment by Calinou** [comment]
> I mean, systems that don't support point size? There must be a lot of AMD users.

Is it an issue on recent AMD graphics cards on Windows though? Billboarded quads have some downsides compared to actual point rendering, and may also perform worse (though this depends on how the hardware implements them).

**Comment by AThousandShips** [comment]
Using the tutorial and creating handle material sets the correct point size on master at least from GDScript

**Comment by Okxa** [comment]
I can reproduce this using the MRP on linux with a Vulkan capable AMD GPU. Same result as shown in this issue before, happens with Forward+ and compability_gl

GPU: AMD RX5700XT
Godot version: `v4.1.2.stable.arch_linux`
Drivers and whatnot from godot output: `OpenGL API 4.6 (Core Profile) Mesa 23.2.1-arch1.2 - Compatibility - Using Device: AMD - AMD Radeon RX 5700 XT (navi10, LLVM 16.0.6, DRM 3.54, 6.5.8-arch1-1)`

![image](https://github.com/godotengine/godot/assets/25027118/841a58c1-8905-4e96-a4ee-1626daadc016)

**Comment by Zireael07** [comment]
Handles likely need similar treatment as origin lines got in #83895

**Comment by FailMelon** [comment]
Make sure you set the point size in the material.


![Godot_v4 1 1-stable_mono_win64_0KEGgj459m](https://github.com/godotengine/godot/assets/3711395/e4d2df05-7ee6-416a-9951-62b960ffb2cf)


https://github.com/godotengine/godot/assets/3711395/5c32b26b-cfa6-486e-8b32-98de399c5155

**Comment by Okxa** [comment]
On further inspection, handles show up in the MRP for me.

However, an addon I made following the 3D gizmo tutorial, does not show the handles when under a specific project:
- I developed the addon under this existing project, and the handles are not visible (now interactable!) when using the addon under this project.
- The handles are shown as normal and are interactable when **the addon is copied to a new project.**

In fact when I make a copy of the original project, and delete `.godot` folder, the handles show up again, but of course introduces some additional work otherwise (Editor UI resets, warnings of some of my (custom) imported files failing to reimport etc.) It would be simpler to solve this some other way.

Is there any specific file that would cache something about this?
